packages/ionic/src/lib/project/index.ts

```diff
@@ -520,7 +521,20 @@ export abstract class Project implements IProject {
   }
 
   async getDistDir(): Promise<string> {
-    return path.resolve(this.directory, 'www');
+    if (this.getIntegration('capacitor') !== undefined) {
+      const conf = new CapacitorConfig(path.resolve(this.directory, CAPACITOR_CONFIG_FILE));
+      const webDir = conf.get('webDir');
+      if (webDir) {
+        return path.resolve(this.directory, webDir);
+      } else {
+        throw new FatalException(
```

Comment:
Would it be better to warn and default to `www` here, instead?

Reply:
well, this I do not know. It depends on how we want the code to behave: if we prefer that people fix the project setting the proper variables or we prefer to make assumptions.
I do not have strong feelings here, but I believe it is more a general product decision.

Reply:
I don't think Capacitor commands will work if there is no webDir configured, it already works by setting it to www as default. So this could only happen if users manually delete it, and will break everything, so I like the FatalException here.